rollup-plugin-size.js

```diff
@@ -21,7 +21,7 @@ const defaults = {
  * @param {boolean} [options.writeFile] option to save filesizes to disk
  */
 function bundleSize(options) {
-  const coreOptions = Object.assign(defaults, options);
+  const coreOptions = Object.assign(options, defaults);
```

Comment:
defaults should be overwritten by options no? 
I think we have to maintain this order `Object.assign(defaults, options)`.

mutating defaults is probably not good
maybe `Object.assign({}, defaults, options)`?